src/client/src/containers/DependencyInfo/index.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import styles from "./styles.module.css";
+import * as getSvg from "../../utils/getSvgUrl";
+import classnames from "classnames";
+import { injectIntl, defineMessages, InjectedIntl } from "react-intl";
+import { WIZARD_CONTENT_INTERNAL_NAMES } from "../../utils/constants";
+import { AppState } from "../../reducers";
+import { IDependenciesInstalled } from "../../reducers/dependencyInfoReducers";
+
+interface IDependencyInfoProps {
+  dependenciesStore: IDependenciesInstalled;
+  frameworkName: string;
+  intl: InjectedIntl;
+}
+
+type Props = IDependencyInfoProps;
+
+const messages = defineMessages({
+  installed: {
+    id: "dependencyChecker.installed",
+    defaultMessage: " detected!"
+  },
+  notInstalled: {
+    id: "dependencyChecker.notInstalled",
+    defaultMessage: " not detected. Click to install."
+  },
+  iconAltMessage: {
+    id: "dependencyChecker.iconAltMessage",
+    defaultMessage: "Icon for dependency checker"
+  }
+});
+
+interface IDependency {
+  commandName: string;
+  dependencyName: string;
+  downloadLink: string;
+  privacyStatementLink: string;
+}
+
+interface IDependencies {
+  [key: string]: IDependency;
+}
+
+const dependencies: IDependencies = {
+  NodeJS: {
+    commandName: "node", // command used on the extension side
```

Comment:
The extension side is still not using this commandName, right?

Reply:
This object is separate from the extension side. 
The thing is that right now I use the payload I get back from the extension directly in the store, so it's lowercase. I think we can discuss how to change it but probably need some mapping from what I send you from the client to the command you actually run on the extension side

Reply:
Ok, hm, what about changing the comment to "message received from the extension side"? because the commandName variable name sounds like it is something that will be executed

Reply:
Yea, naming is hard

Reply:
Talked to @jsondoo about this: 
I will create a map (or just create extra constants) on the extension side for the dependencyName and actual command that will be run. The extension will be checking dependencyName from the client, and also returning dependencyName ("Python" or "Node")